client/cmdhfmfu.h

```diff
@@ -21,7 +21,8 @@ typedef struct {
 	uint8_t	pack[2];
 	uint8_t tbo1[1];
 	uint8_t	signature[32];	
-	//uint8_t	counter[3];
+	uint8_t	counter[3][3];
+	uint8_t pwd[4];
```

Comment:
And authentication key could be  4bytes or 16 (triple des  ULC)

Reply:
You are right. I was only thinking of ev1...